Packs/Fortimail/Integrations/Fortimail/Fortimail.py

```diff
@@ -0,0 +1,2558 @@
+
+    Returns:
+        CommandResults: Outputs of the command that represent an entry in war room.
+    """
+    command_name: str = args.get("command_name", "")
+    command_request_name, command_entity_title, _ = get_command_entity(command_name=command_name)
+    command_request = getattr(client, command_request_name)
+
+    command_args = remove_empty_elements(
+        {
+            "access_control_id": args.get("access_control_id"),
+            "policy_id": args.get("policy_id"),
+            "recipient_policy_id": args.get("recipient_policy_id"),
+            "action": args.get("action"),
+            "reference_id": args.get("reference_id"),
+        }
+    )
+
+    command_request(**command_args)
+
+    return CommandResults(
+        readable_output=command_entity_title,
+    )
+
+
+def list_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """
+    Return the items list according the command entity name.
+
+    Args:
+        client (Client): Session to FortiMail to run API requests.
+        args (dict[str, Any]): Arguments passed down by the CLI to configure the request.
+
+    Returns:
+        CommandResults: Outputs of the command that represent an entry in war room.
+    """
+    command_name: str = args.get("command_name", "")
+    command_request_name, command_entity_title, command_outputs_prefix = get_command_entity(command_name=command_name)
+    # Get the client request function by attribute entity name.
+    command_request = getattr(client, command_request_name)
+    # Get the relevant item key to fetch in case user use the command as GET.
+    # Those arguments cover 22 list commands.
+    command_args = remove_empty_elements(
+        {
+            "group_name": args.get("group_name"),
+            "ip": args.get("ip"),
+            "access_control_id": args.get("access_control_id"),
+            "policy_id": args.get("policy_id"),
+            "recipient_policy_id": args.get("recipient_policy_id"),
+            "email": args.get("email"),
+            "name": args.get("name"),
+            "list_type": args.get("list_type"),
+        }
+    )
+
+    response = command_request(**command_args)
+    # Map the response fields values from integer to string to be informative.
+    output = get_mapped_response(response)
+
+    if not argToBoolean(args.get("all_results")):
+        output = output[: arg_to_number(args.get("limit"))]
+
+    output_table, output = get_updated_response_and_readable_output(output=output, command_args=command_args)
+
+    # Check if command called as list or get command.
+    if not command_args:
+        command_entity_title = f"{command_entity_title} list"
+
+    readable_output = tableToMarkdown(
+        name=command_entity_title,
+        t=remove_empty_elements(output_table),
+        headers=list(output_table[0]) if output_table else [],  # Add ordered headers keys by the updated outputs.
```

Comment:
If `output_table` will be empty as in `None`, `[]` and so on, it will result the `else` condition.
At least from what I see there isn't a scenario where we can receive an index error.

Reply:
You are right thanks for the clarification.